templates/test/tr_job_result_details.html.ep

```diff
@@ -0,0 +1,7 @@
+% if ($comments and $comments > 0) {
```

Comment:
Ah, right. So what is the issue with plain `if ($comments)` again? Am I missing something?